demo/src/screens/componentScreens/ExpandableSectionScreen.tsx

```diff
@@ -1,119 +1,132 @@
+      </View>
+    );
+  };
+
+  renderContent() {
     return (
-      <View margin-10 spread row>
-        <Text grey10 text60>
-          ExpandableSection sectionHeader
+      <View marginH-60>
+        <Text text80>
+          If you have any questions, comments, or concerns, please don&apos;t hesitate to get in touch with us. You can
+          easily reach out to us through our contact form on our website.
+        </Text>
+        <Text text80>
+          Alternatively, you can reach us via email at help@help.com, where our team is ready to assist you promptly. If
+          you prefer speaking with someone directly, feel free to give us a call at 1-833-350-1066.
         </Text>
-        <Image style={styles.icon} source={this.getChevron()}/>
       </View>
     );
   }
 
-  getBodyElement() {
+  renderOptions = () => {
     return (
-      <Carousel>
-        {_.map(this.elements, (element, key) => {
-          return (
-            <View key={key} margin-12>
-              {element}
-            </View>
-          );
-        })}
-      </Carousel>
+      <View margin-page>
+        <Text text70BO marginB-8>
+          Minimum Height
+        </Text>
+        <Text text80 marginB-16>
+          The expandable section can be either fully collapsed, partially expanded to reveal some of the items, or fully
+          expanded by default.
+        </Text>
+        <SegmentedControl
+          activeColor={Colors.$textDefaultLight}
+          activeBackgroundColor={Colors.$backgroundInverted}
+          segments={[{label: 'Default'}, {label: 'Partially'}, {label: 'Fully Expanded'}]}
+          onChangeIndex={index => {
+            switch (index) {
+              case 0:
+                return this.setState({minHeight: DEFAULT});
+              case 1:
+                return this.setState({minHeight: PARTIALLY_EXPANDED_HEIGHT});
+              case 2:
+                return this.setState({minHeight: FULLY_EXPANDED_HEIGHT});
+            }
+          }}
+        />
+      </View>
     );
+  };
+
+  renderExpandableSection = () => {
+    const {expanded, minHeight} = this.state;
+    return (
+      <ExpandableSection
+        top={minHeight === PARTIALLY_EXPANDED_HEIGHT}
+        expanded={expanded}
+        sectionHeader={
+          minHeight === PARTIALLY_EXPANDED_HEIGHT
+            ? this.renderReadMoreHeader()
+            : this.renderHeader('How can I contact you?', expanded, {showInfo: true})
+        }
+        onPress={this.onExpand}
+        minHeight={minHeight}
+      >
+        {this.renderContent()}
+      </ExpandableSection>
+    );
+  };
+
+  renderNextItem() {
+    return this.renderHeader('Where are you located?', false, {disabled: true, showInfo: true});
   }
 
   render() {
-    const {expanded, top} = this.state;
-
+    const {minHeight} = this.state;
     return (
       <ScrollView>
-        <View row center margin-20>
```

Comment:
Missing our usual title "ExpandableSection"

Reply:
Some screens do not have that and we have it in the RNN title

Reply:
I know, but it's strange the the title is "Minimum Height"...